build/README.md

```diff
@@ -64,6 +64,7 @@ Table of Contents
         * [make gen-embedded-openapi](#make-gen-embedded-openapi)
         * [make gen-crd-client](#make-gen-crd-client)
         * [make gen-sdk-grpc](#make-gen-sdk-grpc)
+        * [make gen-allocation-grpc](#make-gen-allocation-grpc)
```

Comment:
`make gen-allocation-grpc` is defined in Makefile, but this was not described in `build/README.md`